### PR TITLE
fix: add disclaimer on user-task-restrictions

### PR DESCRIPTION
**Pull request by nathansandi**
## Description

add a disclaimer on user task restrictions in order to clarify the change takes some time to take effect.

## When should this change go live?

- [x] This is already available but undocumented and should be released within a week.
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by mesellings** [comment]
@nathansandi I cna review this for you 👍

**Review by mesellings — COMMENTED** [review]
I would change this to read as follows (assuming that I have understood correctly, and that the note means that any enable/disable change requires a Tasklist restart):

> Changes to this setting can take a few minutes to be applied, as it requires a Tasklist restart.

Additionally, though this is not the PR scope, I would change the previous sentence as follows:

> User task access restrictions are enabled by default. To disable them, navigate to Console **settings* and turn off the **Enforce user task restrictions** toggle.

**Comment by mesellings** [comment]
@nathansandi I have added some review comments, but as you specifically requested @akeller for the review I have left the approval.

**Comment by akeller** [comment]
> I would change this to read as follows (assuming that I have understood correctly, and that the note means that any enable/disable change requires a Tasklist restart):
> 
> > Changes to this setting can take a few minutes to be applied, as it requires a Tasklist restart.
> 
> Additionally, though this is not the PR scope, I would change the previous sentence as follows:
> 
> > User task access restrictions are enabled by default. To disable them, navigate to Console **settings** and turn off the **Enforce user task restrictions** toggle.

☝️ @nathansandi, please see this feedback. Based on the current wording in the PR, I cannot approve.

**Comment by nathansandi** [comment]
@mesellings @akeller thanks for the reviews :) Just made the suggested updates

File: docs/components/concepts/access-control/user-task-access-restrictions.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ users that belong to `Team A` and the user `example` will have access to the tas
 User task access restrictions are enabled by default on SaaS. For details on user task access restrictions in Self-Managed, visit the [Self-Managed documentation](/self-managed/concepts/access-control/user-task-access-restrictions.md).
 :::
 
-User task access restrictions are enabled by default. To disable them, navigate to the **Settings** section of Console and click the **Enforce user task restrictions** toggle to enable or disable the functionality.
+User task access restrictions are enabled by default. To disable them, navigate to Console settings and turn off the Enforce user task restrictions toggle.
```
Comment (akeller) [inline docs/components/concepts/access-control/user-task-access-restrictions.md:26]:
```suggestion
User task access restrictions are enabled by default. To disable them, navigate to Console settings and turn off the **Enforce user task restrictions** toggle.
```

File: versioned_docs/version-8.5/components/concepts/access-control/user-task-access-restrictions.md [hunk 2]
```diff
@@ -23,7 +23,10 @@ users that belong to `Team A` and the user `example` will have access to the tas
 User task access restrictions are enabled by default on SaaS. For details on user task access restrictions in Self-Managed, visit the [Self-Managed documentation](/self-managed/concepts/access-control/user-task-access-restrictions.md).
 :::
 
-User task access restrictions are enabled by default. To disable them, navigate to the **Settings** section of Console and click the **Enforce user task restrictions** toggle to enable or disable the functionality.
+User task access restrictions are enabled by default. To disable them, navigate to Console settings and turn off the Enforce user task restrictions toggle.
```
Comment (akeller) [inline versioned_docs/version-8.5/components/concepts/access-control/user-task-access-restrictions.md:26]:
```suggestion
User task access restrictions are enabled by default. To disable them, navigate to Console settings and turn off the **Enforce user task restrictions** toggle.
```

**Review by akeller — APPROVED** [review]

**Pull request merged**